packages/webcert/src/App.tsx

```diff
@@ -31,6 +31,7 @@ function App(): JSX.Element {
     window.addEventListener('beforeunload', handleWindowBeforeUnload)
     dispatch(cancelLogout())
     dispatch(getUser())
+    dispatch(getUserTabs())
```

Comment:
Do we need to wait on the data before letting the user interact with the app?

Reply:
No